docker-compose.hobby.yml

```diff
@@ -116,7 +116,6 @@ services:
         restart: on-failure
         environment:
             DATABASE_URL: 'postgres://posthog:posthog@db:5432/posthog'
-            KAFKA_ENABLED: 'true'
```

Comment:
qq: the current release that went out, i.e. `$POSTHOG_APP_TAG` already had kafka_enabled set to true or ignored, correct?